src/Parser/VengeanceDemonHunter/CombatLogParser.js

```diff
@@ -137,33 +144,55 @@ class CombatLogParser extends MainCombatLogParser {
       />,
       <StatisticBox
-        icon={(
+        icon={<Icon icon="spell_mage_altertime" alt="Dead GCD time" />}
+        value={`${formatPercentage(deadTimePercentage)} %`}
+        label='Dead GCD time'
+        tooltip={'Dead GCD time is available casting time not used. This can be caused by latency, cast interrupting, not casting anything (e.g. due to movement/stunned), etc.'}
+      />,
+      <StatisticBox
+      icon={(
           <img
-            src="/img/healing.png"
-            style={{ border: 0 }}
-            alt="Healing"
+          src="/img/healing.png"
+          style={{ border: 0 }}
+          alt="Healing"
           />
-        )}
-        value={`${formatNumber(this.totalHealing / this.fightDuration * 1000)} HPS`}
-        label='Healing done'
-        tooltip={`The total healing done was ${formatThousands(this.totalHealing)}.`}
+      )}
+      value={`${formatNumber(this.totalHealing / this.fightDuration * 1000)} HPS`}
+      label='Healing done'
+      tooltip={`The total healing done was ${formatThousands(this.totalHealing)}.`}
       />,
       <StatisticBox
-        icon={<Icon icon="spell_mage_altertime" alt="Dead GCD time" />}
-        value={`${formatPercentage(deadTimePercentage)} %`}
-        label='Dead GCD time'
-        tooltip='Dead GCD time is available casting time not used. This can be caused by latency, cast interrupting, not casting anything (e.g. due to movement/stunned), etc.'
+        icon={<Icon icon="ability_druid_naturalperfection" alt="Damage absorbed" />}
+        value={`${formatNumber(this.totalDamageTakenAbsorb / this.fightDuration * 1000)} DAPS`}
+        label='Damage absorbed'
+        tooltip={`The total damage absorbed was ${formatThousands(this.totalDamageTakenAbsorb)}.`}
       />,
       <StatisticBox
-        icon={<Icon icon="class_demonhunter" alt="Damage done" />}
-        value={`${formatNumber(this.totalDamageDone / this.fightDuration * 1000)} DPS`}
-        label='Damage done'
-        tooltip={`The total damage done was ${formatThousands(this.totalDamageDone)}.`}
+        icon={<Icon icon="spell_shadow_soulgem" alt="Soul Fragments Generated" />}
+        value={`${formatNumber((this.soulFragmentsCasts / this.fightDuration * 1000) * 60)} SFGPM`}
+        label='Soul Fragments Generated'
+        tooltip={`The total Soul Fragments generated was ${formatThousands(this.soulFragmentsCasts)}.`}
+      />,
+      <StatisticBox
+        icon={<Icon icon="ability_demonhunter_immolation" alt="Immolation Aura Damage" />}
+        value={`${formatNumber(this.immolationAuraDamage / this.fightDuration * 1000)} IADPS`}
+        label='Immolation Aura Damage'
+        tooltip={`The Immolation Aura total damage was ${formatThousands(this.immolationAuraDamage)}.`}
+      />,
+      <StatisticBox
+        icon={<Icon icon="ability_demonhunter_demonspikes" alt="Demon Spikes Casts" />}
+        value={`${formatNumber(this.demonSpikesCasts)} DS`}
+        label='Demon Spikes Casts'
```

Comment:
This is because many spells have different ids for their cast and their buff, damage or heal events. Such as Demon Spikes where the cast is 203720 and the buff is 203819.